zk_toolbox/crates/zk_inception/README.md

```diff
@@ -475,11 +475,21 @@ Initialize prover
 
   Possible values: `gcp`, `local`
 
-## `zk_inception prover generate-sk`
+## `zk_inception prover setup-keys`
 
-Generate setup keys
+Setup keys
 
-**Usage:** `zk_inception prover generate-sk`
+**Usage:** `zk_inception prover setup-keys`
+
+###### **Options:**
+
+- `--mode`
+
+  Possible valuess: `download`, `generate`
+
+- `--region`
```

Comment:
Exactly. If region wasn't specified for generate, it will be working fine, but will throw an error if they weren't for download

Reply:
probably it make sense to add it only for download option

Reply:
I'm not sure how actually... Do clap allow arg to conflict with some specific value of other argument? Anyway, it will affect only the fact that you can use regions flag with generate(and it will do nothing), otherwise in terminal you'll be asked about region only if you chose to download keys

Reply:
it will produce weird help